lib/Dialect/TTNN/IR/TTNNWorkarounds.cpp

```diff
@@ -7,6 +7,7 @@
 #include "ttmlir/Utils.h"
 
 #include "llvm/ADT/SmallVector.h"
+#include <optional>
```

Comment:
Is this needed?

Reply:
You are right, this was added by accident :) Removing it...